core/ppu_bg.go

```diff
@@ -248,10 +271,14 @@ func (ppu *PPU) colorDepth(background uint8) uint8 {
 		switch background {
 		case 0, 1:
 			return 4
+
+		//TODO offset per tile
+		case 2:
```

Comment:
quick question: is it possible to render the background 3 in mode 2 ? I thought it was not according to the schema:

```
// 			1   2   3   4
// ======---=---=---=---=
// 0        4   4   4   4
// 1       16  16   4   -
// 2       16  16   -   -
// 3      256  16   -   -
// 4      256   4   -   -
// 5       16   4   -   -
// 6       16   -   -   -
// 7      256   -   -   -
// 7EXTBG 256 128   -   -
```

Reply:
Not directly but it can be used in a weird mode called offset-per-tile

Reply:
:+1: